### PR TITLE
Fix NOLINTNEXTLINE

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

cc @EikanWang @jgong5 @wenzhe-nrv @sanchitintel

### CODE REVIEW

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 1]
```diff
@@ -213,22 +196,14 @@ std::vector<at::Tensor> constructTensors2(
...
+                             buf_strides_vec[i], buf_dims_vec[i]))
                          .requires_grad(false);
       auto tensor = at::from_blob(
-          // NOLINTNEXTLINE
-          buf_data_vec[i],
-          buf_dims_vec[i],
-          buf_strides_vec[i],
-          options);
+          buf_data_vec[i], buf_dims_vec[i], buf_strides_vec[i], options);
       tensors.emplace_back(tensor);
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:207]:
```suggestion
      tensors.emplace_back(std::move(tensor));
```

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 2]
```diff
@@ -262,11 +232,7 @@ std::vector<at::Tensor> constructTensors2(
         tensors.emplace_back(tensor);
       } else {
         auto tensor = at::from_blob(
-            // NOLINTNEXTLINE
-            buf_data_vec[i],
-            buf_dims_vec[i],
-            buf_strides_vec[i],
-            options);
+            buf_data_vec[i], buf_dims_vec[i], buf_strides_vec[i], options);
         tensors.emplace_back(tensor);
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:236]:
```suggestion
        tensors.emplace_back(std::move(tensor));
```

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 3]
```diff
@@ -152,11 +139,7 @@ std::vector<at::Tensor> constructTensors(
...
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:143]:
```suggestion
        tensors.emplace_back(std::move(tensor));
```

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 4]
```diff
@@ -103,22 +103,14 @@ std::vector<at::Tensor> constructTensors(
...
+                             buf_strides_vec[i], buf_dims_vec[i]))
                          .requires_grad(false);
       auto tensor = at::from_blob(
-          // NOLINTNEXTLINE
-          buf_data_vec[i],
-          buf_dims_vec[i],
-          buf_strides_vec[i],
-          options);
+          buf_data_vec[i], buf_dims_vec[i], buf_strides_vec[i], options);
       tensors.emplace_back(tensor);
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:114]:
```suggestion
      tensors.emplace_back(std::move(tensor));
```

**Review by Skylion007 — APPROVED** [review]

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 5]
```diff
@@ -152,12 +139,8 @@ std::vector<at::Tensor> constructTensors(
         tensors.emplace_back(tensor);
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:139]:
```suggestion
        tensors.emplace_back(std::move(tensor));
```
@cyyever Missed one

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 6]
```diff
@@ -103,23 +103,15 @@ std::vector<at::Tensor> constructTensors(
...
                          .layout(at::kStrided)
                          .device(at::kCPU) // TODO: support GPUs too
                          .memory_format(deduce_memory_format(
-                             // NOLINTNEXTLINE
-                             buf_strides_vec[i],
-                             // NOLINTNEXTLINE
-                             buf_dims_vec[i]))
+                             buf_strides_vec[i], buf_dims_vec[i]))
                          .requires_grad(false);
       auto tensor = at::from_blob(
```
Comment (Skylion007) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:112]: FYI, @cyyever this is bad code smell, and we should probably emplace the tensors from factory ctor in the future so they can be better elided.

Comment (cyyever) [reply torch/csrc/jit/tensorexpr/external_functions.cpp:112]: Given  the current state of JIT code and new alternatives, we may not need to change them so thoroughly.

**Pull request closed without merging**